.github/workflows/wheel_macos_x86_64.yml

```diff
@@ -49,7 +49,8 @@ jobs:
           key: ${{ matrix.os }}-kokkos${{ matrix.kokkos_version }}-${{ matrix.exec_model }}
 
       - name: Install clang
-        run: brew install llvm
+        run: |
+          brew install libomp
```

Comment:
Unlike `llvm`, there is no need to install g++, right?

Reply:
Nope, as above, maps to the MacOS clang compiler internally.